js/algoliasearch/click_conversion_analytics.js

```diff
@@ -1,89 +1,108 @@
 algoliaBundle.$(function ($) {
 	AlgoliaAnalytics.init({
-		applicationID: algoliaConfig.applicationId,
+		appId: algoliaConfig.applicationId,
 		apiKey: algoliaConfig.instant.apiKey
 	});
 
 	// "Click" in autocomplete
 	$(algoliaConfig.autocomplete.selector).each(function () {
-		$(this).on('autocomplete:selected', function (e, suggestion) {
-			trackClick(suggestion.objectID, suggestion.__position, suggestion.__queryID);
+		$(this).on('autocomplete:selected', function (e, suggestion, dataset, context) {
+
+			var sources = analyticsHelper.sources;
+			var source = sources.find(function(e) {
+				return e.name == dataset;
+			});
```

Comment:
1. does this still work if I use the twice the same dataset?
2. [find](https://caniuse.com/#feat=array-find) is not supported on IE11, it's best to use the _.find or use [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter).